client/my-sites/stats/stats-module/style.scss

```
@@ -570,3 +570,23 @@ ul.module-header-actions {
 .stats-module__header.is-refreshing {
 	animation: loading-fade 1.6s ease-in-out infinite;
 }
+
+.stats-module__availability-warning {
+	padding: 12px 20px 0;
+	margin-bottom: -18px;
```

We can remove the `margin-bottom` on the `.stats-module__availability-warning-message` instead.

```suggestion
	.stats-module__availability-warning-message {
		margin-bottom: 0;
	}
```

---

Thanks - that's a nicer solution!